package.json

```diff
@@ -26,7 +26,7 @@
     "file-saver": "^2.0.5",
     "filesize": "^7.0.0",
     "github-markdown-css": "^4.0.0",
-    "highcharts": "^9.3.3",
+    "highcharts": "^10.3.3",
```

Comment:
This is a major version upgrade because 10.0.0 is where the `bypassHTMLFiltering` feature was added, so I upgraded to the most recent 10.x.x. The changes in 10.0.0 ([link](https://www.highcharts.com/changelog/#highcharts-v10.0.0)) don't seem to affect our chart, and I manually checked that different parts of the spend report still work as before.